sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/AzureMonitorTransmitterTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.Exporter.Tests.CommonTestFramework;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
+{
+    public class AzureMonitorTransmitterTests
+    {
+        [Theory]
+        [InlineData("InstrumentationKey=001", null, "001")] // Options Only
+        [InlineData(null, "InstrumentationKey=002", "002")] // Env Var Only
+        [InlineData("InstrumentationKey=001", "InstrumentationKey=002", "001")] // If both set, Options will be used.
+        public void Verify_InitializeConnectionVars(string? optionsConnStr, string? envVarConnStr, string expectedIkey)
+        {
+            var options = new AzureMonitorExporterOptions();
+            if (optionsConnStr != null)
+            {
+                options.ConnectionString = optionsConnStr;
+            }
+
+            var platform = new MockPlatform();
+            if (envVarConnStr != null)
+            {
+                platform.SetEnvironmentVariable("APPLICATIONINSIGHTS_CONNECTION_STRING", envVarConnStr);
```

Comment:
We should consider cleaning up environment variable.  It could product inconsistent test results once more tests are added.

Reply:
That's the whole point of the platform abstractions :)
Here the MockPlatform instance will be disposed at the end of the test without impacting any other test.

Reply:
I see, it is added to dictionary.